certified-connectors/PowerAppsCards/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1970 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CardsforPowerApps",
```

Comment:
@GarimaSarangal since you are an internal Microsoft Partner, we can allow using our products name in the title. can you please make the title as "Cards for PowerApps"? thank you.

Reply:
updated.